build_resume.js

```diff
@@ -0,0 +1,36 @@
+const readresume = require('./read-schema')
+
+const buildResume = function (resumeObject) {
+  const parsedData = JSON.parse(resumeObject)
+  const basics = parsedData["basics"];
+  const name = basics["name"];
+  const label = basics["label"];
+  const email = basics["email"];
```

Comment:
can use object destructuring to make it more concise, which looks something like this:
```javascript
const {name, label, email, ...rest} = basics;
```

(UNTESTED)